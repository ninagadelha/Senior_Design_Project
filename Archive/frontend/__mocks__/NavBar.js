import React from 'react';

// const MockNavBar = jest.fn(() => <div data-testid="mock-navbar">Mock NavBar</div>);

const MockNavBar = () => {
    <div data-testid="mock-navbar">Mock NavBar</div>
  };

export default MockNavBar;
