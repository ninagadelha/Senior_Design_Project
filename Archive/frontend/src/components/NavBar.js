import React from "react";
import { ReactComponent as Logo } from "./logo512.png";

const NavBar = () => {
  return (
    <>
      <Nav>
        <NavLogo to="/">
          Logo
          <Logo />
        </NavLogo>

        <Bars />

        <NavMenu>
          <NavLink to="/home/" activeStyle={{ color: "black" }}>
            Home
          </NavLink>
          <NavLink to="/survey/" activeStyle={{ color: "black" }}>
            Take Survey
          </NavLink>
          <NavLink to="/results/" activeStyle={{ color: "black" }}>
            View Results
          </NavLink>
          <NavBtn>
            <NavBtnLink to="/login/">Login</NavBtnLink>
          </NavBtn>
        </NavMenu>
      </Nav>
    </>
  );
};
export default NavBar;
