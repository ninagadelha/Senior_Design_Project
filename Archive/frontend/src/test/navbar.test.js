import renderer from "react-test-renderer";
import NavBar from "../routes/navbar";

jest.mock("../styles/navbar.css");
jest.mock("react-router-dom");

const renderTree = (tree) => renderer.create(tree);
describe("<NavBar>", () => {
  it("should render component", () => {
    expect(renderTree(<NavBar />).toJSON()).toMatchSnapshot();
  });
});
