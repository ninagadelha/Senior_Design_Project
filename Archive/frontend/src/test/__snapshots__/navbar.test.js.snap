// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar> should render component 1`] = `
<nav
  className="navbar"
>
  <div
    className="navbar__container"
  >
    <div
      className="navbar__toggle"
      id="mobile-menu"
      onClick={[Function]}
    >
      <span
        className="bar"
      />
       
      <span
        class="bar"
      />
      <span
        className="bar"
      />
    </div>
    <ul
      className="navbar__menu"
    >
      <li
        className="navbar__item"
      />
      <li
        className="navbar__item"
      />
      <li
        className="navbar__item"
      />
      <li
        className="navbar__item"
      />
    </ul>
  </div>
</nav>
`;
